CHANGELOG.md

```diff
@@ -49,6 +50,25 @@ This release depends on, and has been tested with, the following Mapbox dependen
 - Mapbox Java `v6.8.0` ([release notes](https://github.com/mapbox/mapbox-java/releases/tag/v6.8.0))
 - Mapbox Android Core `v5.0.2` ([release notes](https://github.com/mapbox/mapbox-events-android/releases/tag/core-5.0.2))
 
+To migrate change your code from:
+```kotlin
+binding.navigationView.customizeMapView(customMapView)
+```
+into:
+```kotlin
+class CustomMapViewBinder: MapViewBinder() {
+
+  override fun getMapView(viewGroup: ViewGroup): MapView = customMapView.also {
+    viewGroup.removeAllViews()
+    viewGroup.addView(it)      
```

Comment:
Good catch. Also, I think we can shorten this code snippet to just
```kotlin
// old
navigationView.customizeMapView(customMapView)

// new 
navigationView.customizeViewBinders {
    mapViewBinder = object : MapViewBinder() {
         override fun getMapView(viewGroup: ViewGroup): MapView = customMapView
    }
}
```
Btw. this brought one more thing to my attention.
If `customMapView` already has a parent `customMapView.getParent() != null` the `viewGroup.addView(customMapView, ...)` call will [throw `IllegalStateException`](https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/view/ViewGroup.java#5247)

Reply:
> If customMapView already has a parent customMapView.getParent() != null the viewGroup.addView(customMapView, ...) call will [throw IllegalStateException](https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/view/ViewGroup.java#5247)

That's right but I consider that it's expected. Why pass a View that has a parent if it's going to be added to another parent? Maybe I'll mention it in docs?

Reply:
This is already mentioned in the public docs. As a developer, when you are creating a custom binder it will be added to `FrameLayout`(in most cases) by default. https://docs.mapbox.com/android/navigation/guides/drop-in-ui/customize-drop-in-ui/#define-the-layout-for-custom-view